bread/bread.py

```diff
@@ -163,7 +163,7 @@ def get_form(self, data=None, files=None, **kwargs):
             self.form_class = modelform_factory(
                 self.bread.model,
                 fields='__all__',
-                exclude=self.exclude
+                exclude=self.bread.exclude
```

Comment:
This fixes a test failure (run tests by running "tox").  Philip also ran into it when he started using bread for some new views. He included this fix in #15 anyway, so this change is already in develop.